src/services/repo/useRepoFlagsSelect.test.tsx

```diff
@@ -344,9 +317,6 @@ describe('FlagsSelect', () => {
           wrapper: pullWrapper(),
         })
 
-        await waitFor(() => result.current.isFetching)
-        await waitFor(() => !result.current.isFetching)
```

Comment:
Have you seen any flakiness after removing these? Sometimes they seem to resolve erroneous failures for me

Reply:
I haven't noticed any issues, removing it. Throughout the migration i was trying to remove extra promises to reduce the workload that is going on. If we find that they're flaking more, we can re-introduce them.

Reply:
Cool sg